rust_src/src/lib.rs

```diff
@@ -1,23 +1,24 @@
 #![cfg_attr(feature = "clippy", feature(plugin))]
 #![cfg_attr(feature = "clippy", plugin(clippy))]
-#![cfg_attr(feature = "clippy", allow(not_unsafe_ptr_arg_deref, wrong_self_convention))]
+#![cfg_attr(
+    feature = "clippy",
+    allow(not_unsafe_ptr_arg_deref, wrong_self_convention)
+)]
 #![feature(const_fn)]
-#![feature(const_size_of)]
 #![allow(non_upper_case_globals)]
 #![allow(non_snake_case)]
 #![allow(private_no_mangle_fns)]
 #![allow(non_camel_case_types, non_snake_case, non_upper_case_globals)]
```

Comment:
Maybe move this comment before the allow instead of allowing it to be placed after?

Reply:
Fixed